turborepo-tests/integration/tests/command-ls.t

```diff
@@ -0,0 +1,37 @@
+Setup
+  $ . ${TESTDIR}/../../helpers/setup_integration_test.sh
+
+Run info
+  $ ${TURBO} ls
+   WARNING  ls command is experimental and may change in the future
+  3 packages
+  
+    another packages/another
+    my-app apps/my-app
+    util packages/util
+
+Run info with filter
+  $ ${TURBO} ls -F my-app...
+   WARNING  ls command is experimental and may change in the future
+  2 packages
+  
+    my-app apps/my-app
+    util packages/util
+
+Run info on package `another`
+  $ ${TURBO} ls another
```

Comment:
This was interesting/surprising. I almost expected to see tasks with the `-F` flag, but I kind of understand only showing them for a single package at a time and `-F` is not just for a single package. I know there discussion of a `-a` flag as an alternative. I'm curious to see what others think!